packaging/neo4j-desktop/src/main/resources/org/neo4j/desktop/config/neo4j-default.conf

```diff
@@ -10,32 +10,39 @@ dbms.directories.import=import
 # Require (or disable the requirement of) auth to access Neo4j
 dbms.security.auth_enabled=true
 
-#
+# With default configuration Neo4j only accepts local connections.
+# To accept non-local connections, uncomment this line:
+#dbms.connectors.default_listen_address=0.0.0.0
+
+# You can also choose a specific network interface, and configure a non-default
+# port for each connector, by setting their individual listen_address.
+
+# Neo4j advertises the addresses its connectors.
+# To specify the hostname or IP address of this server used in those
+# adverised addresses, uncomment this line:
+#dbms.connectors.default_advertised_hostname=localhost
```

Comment:
The "uncomment this line" approach is reserved for cases where the line _doesn't_ contain the default. I think it does here, doesn't it?

Also "adverised".